tests/e2e/normalize.go

```diff
@@ -129,6 +131,13 @@ func normalizeKRMObject(u *unstructured.Unstructured, project testgcp.GCPProject
 				})
 			}
 		}
+
+		switch u.GroupVersionKind() {
```

Comment:
Do you have suggestions on how we want to maintain a custom list in the long term? Personally I like listing individual kinds in tests like this one and use generic methods in the code. But then I realize recently that it may be uneasy to discover for contributors.

Reply:
Interesting question.  One idea is that we could just replace spec.resourceID with `"${" + Kind + "ResourceID}"` - that would probably give good results.  It's hard to know while we have only have one example here though.